src/EnvironmentChecker.php

```diff
@@ -237,8 +257,19 @@ public function log($message, $level)
     public function setErrorCode($errorCode)
     {
         $this->errorCode = $errorCode;
+
+        return $this;
     }
 
+
+    public function setIncludeDetails($includeDetails)
```

Comment:
Can we also add at least a simple docblock?

Reply:
Done